packages/mobile/scripts/unlock.sh

```diff
@@ -18,7 +18,13 @@ done
 adb wait-for-device shell \
   'while [[ -z $(getprop sys.boot_completed) ]]; do sleep 1; done;'
 
+
+echo "locksettings set-pin 123456" | adb shell
+
+sleep 1
 echo "Device is done booting"
+sleep 10
```

Comment:
`adb wait-for-device` would be better here.

Reply:
Just to make sure the device is responding to keyboard events before trying to unlocking